src/lsptoolshost/dotnetRuntimeExtensionResolver.ts

```diff
@@ -88,18 +88,17 @@ export class DotnetRuntimeExtensionResolver implements IHostExecutableResolver {
     /**
      * Acquires the .NET runtime and any other dependencies required to spawn a particular .NET executable.
      * @param path The path to the entrypoint assembly. Typically a .dll.
-     * @returns The path to the `dotnet` command to use to spawn the process.
      */
-    private async acquireDotNetProcessDependencies(path: string) {
-        const dotnetPath = await this.acquireRuntime();
+    private async acquireDotNetProcessDependencies(path: string): Promise<HostExecutableInformation> {
```

Comment:
The implicit return type didn't match the docs, but this is what it's returning. (And that's what other code is expecting.)